storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -661,8 +661,8 @@
           storm-conf-file (System/getProperty "storm.conf.file")
           storm-log-dir (or (System/getProperty "storm.log.dir") (str storm-home file-path-separator "logs"))
           storm-conf (read-storm-config)
-          storm-log-conf-dir (storm-conf "storm.logback.conf.dir")
-          storm-logback-conf-dir (or storm-log-conf-dir (str storm-home file-path-separator "log4j2"))
+          storm-log-conf-dir (storm-conf "storm.log4j2.conf.dir")
```

Comment:
Hey, there. I know you're just updating the existing code, but would you mind adding `"storm.log4j2.conf.dir"` to Config.java and removing the `storm-conf (read-storm-config)` binding?

Both of those lines were added in #373 and went uncaught. Instead of reading from `storm-conf`, just read from `conf`.